lib/ruby/truffle/mri/yaml.rb

```diff
@@ -0,0 +1 @@
+require_relative '../../stdlib/' + File.basename(__FILE__)
```

Comment:
Could we just have a symlink? I guess that might be a problem with the jar though.